SimplenoteTests/NoticeContorllerTests.swift

```diff
@@ -1,33 +1,142 @@
-//
-//  NoticeContorllerTests.swift
-//  SimplenoteTests
-//
-//  Created by Charlie Scheer on 3/15/21.
-//  Copyright © 2021 Automattic. All rights reserved.
-//
-
 import XCTest
+@testable import Simplenote
 
 class NoticeContorllerTests: XCTestCase {
+    private lazy var presenter = MockNoticePresenter()
+    private lazy var controller = NoticeController(presenter: presenter)
+
+func testSetupNoticeContoller() {
+    controller.setupNoticeController()
+
+    XCTAssertTrue(presenter.listeningToKeyboard == true)
+    }
+
+    func testPresentNotice() throws {
+        let notice = Notice(message: "Message", action: nil)
+        controller.present(notice, withTimer: NoticeContorllerTests.noActionTimer)
+
+        XCTAssertTrue(presenter.presented)
+
+        let presentedNoticeView = try XCTUnwrap(presenter.noticeView)
+        XCTAssertEqual(presentedNoticeView.message, notice.message)
+        XCTAssertTrue(presentedNoticeView.handler == nil)
```

Comment:
This is the same as in `testMakeNoticeView`. Because they're testing the same thing, we can remove checks from this method.